Content.Server/Botany/Systems/PlantHolderSystem.cs

```diff
@@ -262,16 +269,15 @@ private void OnInteractUsing(Entity<PlantHolderComponent> entity, ref InteractUs
                 return;
             }
 
-            component.Health -= (_random.Next(3, 5) * 10);
-
-            if (!component.Harvest)
+            if ((int) (component.Age * component.Seed.GrowthStages / component.Seed.Maturation)<=1)
```

Comment:
As before, please note whitespace conventions.

Also, since this calculation is not super obvious, it may be better to factor out the common computation into a function (e.g. `GetCurrentGrowthStage()`) instead of copy/paste.